### PR TITLE
cpu/stm32l4: optmized isr vector definitions

### PR DESCRIPTION
**Pull request by haukepetersen**
follow up on #7535

### CODE REVIEW

File: cpu/stm32l4/vectors.c [hunk 1]
```diff
@@ -18,236 +18,185 @@
...
+    [49] = isr_sdmmc1,               /* [49] SDMMC1 global Interrupt */
+    [50] = isr_tim5,                 /* [50] TIM5 global Interrupt */
+    [52] = isr_uart4,                /* [52] UART4 global Interrupt */
+    [53] = isr_uart5,                /* [53] UART5 global Interrupt */
+    [61] = isr_dfsdm1_flt0,          /* [61] DFSDM1 Filter 0 global Interrupt */
+    [62] = isr_dfsdm1_flt1,          /* [62] DFSDM1 Filter 1 global Interrupt */
+    [63] = isr_dfsdm1_flt2,          /* [63] DFSDM1 Filter 2 global Interrupt */
+    [67] = isr_otg_fs,               /* [67] USB OTG FS global Interrupt */
+    [75] = isr_sai2,                 /* [75] Serial Audio Interface 2 global interrupt */
+    [78] = isr_lcd,                  /* [78] LCD global interrupt */
```
Comment (aabadie) [inline cpu/stm32l4/vectors.c]: 2 interrupts are missing for stm32l476rg:
```
[80] = isr_rng,
[81] = isr_fpu,
```

Comment (haukepetersen) [reply cpu/stm32l4/vectors.c]: hm, thats bad. Need to look into this

Comment (haukepetersen) [reply cpu/stm32l4/vectors.c]: Looked into this: they are not missing. Both vectors are defined in the common section above.

**Comment by haukepetersen** [comment]
rebased

**Comment by aabadie** [comment]
please squash

**Comment by haukepetersen** [comment]
rebased and squashed.

**Comment by aabadie** [comment]
All green, go !

**Pull request merged**